modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/macros/SchemaAPI.macro

```diff
@@ -42,9 +42,14 @@ definition {
 	macro createAPISchema {
 		Variables.assertDefined(parameterList = "${mainObjectDefinitionERC},${name},${externalReferenceCode}");
 
+		if (!(isSet(externalReferenceCodeSchema))) {
```

Comment:
Could you rename to schemaExternalReferenceCode?

Reply:
I checked we also named the ERC like this before, so I think it's better to use this